src/frontend.rs

```diff
@@ -746,6 +752,32 @@ impl AsyncComponent for App {
                     }
                 }
             });
+            let input_sender = sender.input_sender().clone();
+
+            sender.command(move |_sender, shutdown_receiver| {
+                shutdown_receiver
+                    .register(async move {
```

Comment:
You should use `.spawn_command()` instead for blocking tasks or you'll be blocking executor with fully synchronous code below.